packages/testing/src/api/index.ts

```diff
@@ -64,6 +67,34 @@ export function initializeTestApp(options: any): admin.app.App {
   );
 }
 
+export function initializeFirestoreTestApp(options: any): FirebaseApp {
+  if (!('projectId' in options)) {
+    throw new Error('projectId not specified');
+  }
+  if (typeof options.auth != 'object') {
+    throw new Error('auth must be an object');
+  }
```

Comment:
If you want, you could declare the type for `options` (instead of leaving it as `any`) and let the compiler construct these assertions for you.

Reply:
done